lib/mongo/collection.rb

```diff
@@ -239,6 +253,13 @@ def create(opts = {})
       options = Hash[self.options.reject do |key, value|
         %w(read read_preference read_concern).include?(key.to_s)
       end]
+      options.update(opts.slice(*TIME_SERIES_OPTIONS.keys))
+      # Converting Ruby spelled time series options to server style.
+      TIME_SERIES_OPTIONS.each do |ruby_key, server_key|
+        if options.has_key?(ruby_key)
```

Comment:
We generally use `key?` instead of `has_key?` but you don't need to change this now.